DFC.Composite.Shell/Services.Application/ApplicationService.cs

```diff
@@ -143,7 +143,7 @@ private Task<string> GetPostMarkUpAsync(ApplicationModel application, string pat
             }
 
             var uri = new Uri(bodyRegion.RegionEndpoint);
-            var url = $"{uri.Scheme}://{uri.Authority}/{article}";
+            var url = $"{uri.Scheme}://{uri.Authority}/{path}/{article}";
```

Comment:
This was actually a deliberate change, since a post action would come from '/dysac/q/01' but the underlying route on the target system should only be '/q/01' which is just the article. The underlying system doesn't know anything about the '/dysac' route.

Reply:
There might be a conflict here then as the other apps POST uses the path to route the post inside the child app.
Are you able to map any incoming /dysac routes and remove the leading "/dysac" in your child app?

Reply:
Humm.. If I remove `/dysac` from the POST Url on the form then I end up with the a 404, since it will try and post to `https://localhost:5001/q/01` instead of `https://localhost:5001/dysac/q/01` (or a direct POST to `https://localhost:5003/q/01`). If the path is included then I end up with `https://localhost:5003/dysac/q/01` which is incorrect because the child app doesn't have the `dysac` route. Just so I'm clear are children apps you have aware of there composite route.

Reply:
aha this might be a registration issue and a miss-understanding on both sides of the fence. What is the endpoint in your region registration for the Body region? does it have "dysac" in the path?

Reply:
No, it is just `http://localhost:5003` no `dysac` path, as that doesn't exist in the `dysac` routing.

Reply:
OK I will look into it as we have a clash of requirements here.

Reply:
👍

Reply:
OK I've had a look at this and I think we need a switch to include the path in the url or not. Best to add a boolean to the Path registration (IncludePathInPostUrl) which for Dysac will be false, and all others will be true.
Any thoughts?